doc/getting_started/index.rst

```diff
@@ -471,71 +471,154 @@ Install a Toolchain
 
          .. note::
-            It is recommended to install the Zephyr SDK at one of the following locations:
+            It is recommended to extract the Zephyr SDK bundle at one of the following locations:
+
+            * ``$HOME``
+            * ``$HOME/.local``
+            * ``$HOME/.local/opt``
+            * ``$HOME/bin``
+            * ``/opt``
+            * ``/usr/local``
 
-            * ``$HOME/zephyr-sdk[-x.y.z]``
-            * ``$HOME/.local/zephyr-sdk[-x.y.z]``
-            * ``$HOME/.local/opt/zephyr-sdk[-x.y.z]``
-            * ``$HOME/bin/zephyr-sdk[-x.y.z]``
-            * ``/opt/zephyr-sdk[-x.y.z]``
-            * ``/usr/zephyr-sdk[-x.y.z]``
-            * ``/usr/local/zephyr-sdk[-x.y.z]``
+            The Zephyr SDK bundle archive contains the ``zephyr-sdk-0.14.0`` directory and, when
+            extracted under ``$HOME``, the resulting installation path will be
+            ``$HOME/zephyr-sdk-0.14.0``.
 
-            where ``[-x.y.z]`` is optional text, and can be any text, for example ``-0.13.2``.
+      #. Run the Zephyr SDK bundle setup script:
 
-            If installing the Zephyr SDK outside any of those locations, please read: :ref:`zephyr_sdk`
+         .. code-block:: bash
 
-            You cannot move the SDK directory after you have installed it.
+            cd zephyr-sdk-0.14.0
+            ./setup.sh
+
+         .. note::
+            You only need to run the setup script once after extracting the Zephyr SDK bundle.
+
+            You must rerun the setup script if you relocate the Zephyr SDK bundle directory after
+            the initial setup.
 
       #. Install `udev <https://en.wikipedia.org/wiki/Udev>`_ rules, which
          allow you to flash most Zephyr boards as a regular user:
 
          .. code-block:: bash
 
-            sudo cp ~/zephyr-sdk-0.13.2/sysroots/x86_64-pokysdk-linux/usr/share/openocd/contrib/60-openocd.rules /etc/udev/rules.d
+            sudo cp ~/zephyr-sdk-0.14.0/sysroots/x86_64-pokysdk-linux/usr/share/openocd/contrib/60-openocd.rules /etc/udev/rules.d
             sudo udevadm control --reload
 
    .. group-tab:: macOS
 
-      Follow the instructions in :ref:`gs_toolchain`. Note that the Zephyr SDK
-      is not available on macOS.
+      #. Download and verify the `latest Zephyr SDK bundle
+         <https://github.com/zephyrproject-rtos/sdk-ng/releases>`_:
 
-      Do not forget to set the required :ref:`environment variables <env_vars>`
-      (:envvar:`ZEPHYR_TOOLCHAIN_VARIANT` and toolchain specific ones).
+         .. code-block:: bash
+
+            cd ~
+            wget https://github.com/zephyrproject-rtos/sdk-ng/releases/download/v0.14.0/zephyr-sdk-0.14.0_macos-x86_64.tar.gz
+            wget -O - https://github.com/zephyrproject-rtos/sdk-ng/releases/download/v0.14.0/sha256.sum | shasum --check --ignore-missing
+
+         If your host architecture is 64-bit ARM (Apple Silicon, also known as M1), replace
```

Comment:
I'd personally remove M1 references, I'm pretty sure we'll have M2 or M1plus or whatever at some point.

Reply:
I think just mentioning `Apple Sillicon` means we don't have to keep updating the document when, e.g., M2 is released.

Reply:
I agree with @gmarull, let's stick to "Apple Silicon". Please send a follow-up PR @stephanosio.

Reply:
Looking at https://www.apple.com/mac-mini/, they almost exclusively refer to their Arm-based Mac Minis as "M1" and there is not a single mention of the "Apple Silicon" on that page (from what I can see, they stopped using the "Apple Silicon" term, at least in the marketing materials); in that sense, I think it is helpful to keep that "aka. M1" for now.

When they release a new M series SoC, I will make sure to send a follow-up PR either generalising that phrase or just completely removing it.